model/post.go

```diff
@@ -665,3 +665,7 @@ func RewriteImageURLs(message string, f func(string) string) string {
 
 	return string(result)
 }
+
+func (p *Post) IsFromOAuthBot() bool {
+	return p.GetProp("from_webhook") == "true" && p.GetProp("override_username") != ""
```

Comment:
Since we are checking multiple props I'd suggest to use `p.GetProps()` once and check on the returned value.
This is mainly to avoid additional locking since post props are guarded by a mutex.